templates/navButton.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { classes, compile } from 'core/js/reactHelpers';
+
+export default function NavButton(props) {
+  const {
+    text,
+    _iconClasses
+  } = props;
+  return (
+    <React.Fragment>
```

Comment:
Not sure what the preference here is but we could use the newer syntax `<></>` here over `<React.Fragment></React.Fragment>`